elements/lisk-p2p/test/integration/p2p.ts

```diff
@@ -2110,9 +2110,6 @@ describe('Integration tests for P2P library', () => {
 					(n: Number[]) => n.includes(5003) || n.includes(5004),
 				);
 			});
-
-			it.skip('should not evict peers with low latency', async () => {});
```

Comment:
Why do we remove them? Are we implementing these cases in another place? Are they not relevant anymore?

Reply:
Firstly they do not belong in this describe block as the config is only for `connectTime`. Secondly yes these should be tested outside of integration as it's not possible to fake high ping node in integration.